docs/sources/tempo/release-notes/v2-6.md

```diff
@@ -168,7 +168,11 @@ vParquet4 format is now the default block format.
 It's production ready and we highly recommend switching to it for improved query performance. [PR [3810](https://github.com/grafana/tempo/pull/3810)]
 
 Upgrading to Tempo 2.6 modifies the Parquet block format.
-Although you can use Tempo 2.6 with vParquet2 or vParquet3, you can only use Tempo 2.6 with vParquet3.
+You don't need to do anything with Parquet to go from 2.5 to 2.6.
+All of your old blocks remain vParquet3, which 2.6 reads.
```

Comment:
The default version of vParquet in Tempo 2.5 was vParquet3 but it is possible some users chose to use vParquet2

Reply:
```suggestion
If you used vParquet2 or vParquet3, all of your old blocks remain and are readable by version 2.6.
```